py/surveysim/afternoonplan.py

```diff
@@ -158,24 +160,37 @@ def afternoonPlan(self, day_stats, tiles_observed):
         for i in range(len(self.tileID)):
             if ( self.status[i] < 2 ):
                 # Add this tile to the plan, first adjust its priority.
-                if self.program[i] == 'DARK':
+                if (self.obsconds[i] == 1 or self.obsconds[i] == 2): # Pass 1, 2, 3, & 4 are numbered 0, 1, 2, 3.
```

Comment:
Rather than hardcoding 1 and 2, this should use code like
```python
from desitarget import obsconditions as obsbits
if (self.obsconds[i] & obsbits.mask('DARK|GRAY')) != 0:
    ...
```
i.e. use the bit definitions from desitarget, and test if the bit is set rather than if the obsconds is exactly equal to a single number.  In the future, more than one bit might be set for a tile, e.g. to allow a tile to be observed in either BRIGHT or TWILIGHT18 conditions.